android/src/main/java/com/jaumard/smsautofill/SmsAutoFillPlugin.java

```diff
@@ -56,13 +57,11 @@ private SmsAutoFillPlugin(MethodChannel channel, Registrar registrar) {
 
             @Override
             public boolean onActivityResult(int requestCode, int resultCode, Intent data) {
-                if (requestCode == PHONE_HINT_REQUEST && pendingHintResult != null) {
-                    if (resultCode == Activity.RESULT_OK && data != null) {
```

Comment:
by removing those checks we will now get NPE exceptions

Reply:
When result code is Activity.RESULT_OK, data is not null.

Reply:
you are not sure it 100% the case of all brand and manufacturer in fact :) and `pendingHintResult` can be null

Reply:
I got it. Thank you.